### DISCUSSION TITLE
I applied github student developer pack on 31st May but until now . I have't got any mail . They say that your request was approved and your benefits will be available soon.

### DISCUSSION THREAD

**Discussion by ThutaNyan788** [community/community#127058] [GitHub Education]
### Hi everyone,

Please help me how to solve this problem

**Comment by davevad93** [comment]
Hi @ThutaNyan788 ,

That's not an issue, it's the default procedure, if your request has been approved it displays that message with a green label that states `Approved`. When your benefits are fully available that label will turn purple and the message will be "`Congrats! Your request for @ThutaNyan788 was approved and your benefits are now available.`". Be patient because the waiting time can vary from a few days to a few weeks.

**Reply by ThutaNyan788** [reply]
Thanks @davevad93

**Reply by davevad93** [reply]
You're welcome @ThutaNyan788 !

**Reply by ThutaNyan788** [reply]
@davevad93 how long GitHub student developer pack lifetime ?

**Reply by davevad93** [reply]
Hey @ThutaNyan788 , the student validation lasts for 2 years, after which you will need to revalidate your student status, so in 2 years if you'll still have student status you can reapply for your benefits.
